lib/jekyll-relative-links/generator.rb

```diff
@@ -78,7 +114,7 @@ def url_for_path(path)
     end
 
     def potential_targets
-      @potential_targets ||= (site.pages + site.static_files)
+      @potential_targets ||= (site.pages + site.static_files + site.docs_to_write)
```

Comment:
We should probably note that all documents are eligible to be targets... it's only a matter of opting in to them being processed for rewrites.